coderd/workspaceagents_test.go

```diff
@@ -1053,6 +1055,87 @@ func TestWorkspaceAgentListeningPorts(t *testing.T) {
 	})
 }
 
+func TestWorkspaceAgentContainers(t *testing.T) {
```

Comment:
Vibe: seems a bit redundant to have this test in both layers
I assume this is because it's easier than starting up an agent and a tailnet to test the full integration?

Reply:
It does actually start an agent and tailnet! This tests the end-to-end flow of client -> coderd -> agent -> docker ps -> agent -> coderd -> client.

Reply:
Aha!